File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.c
int maximumCount(int* nums, int numsSize) {
    int i, negCount = 0; 
    for (i = 0; i < numsSize; i++) {
        if (nums[i] < 0) {
            negCount++;
        }
        if (nums[i] > 0) {
            break;
        }
    }
    return negCount > (numsSize - i) ? negCount : (numsSize - i);
}

File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maximumCount = function(nums) {
    let negCount = 0;
    let i;

    for (i = 0; i < nums.length; i++) {
        if (nums[i] < 0) {
            negCount++;
        }
        if (nums[i] > 0) {
            break;
        }
    }

    return Math.max(negCount, nums.length - i);
};